bridge_adaptivity/.codeclimate.yml

```diff
@@ -0,0 +1,50 @@
+# Serves for configuration of CodeClimate (automated code review for test coverage, complexity, duplication, etc.)
```

Comment:
This config file is redundant remove it please

Reply:
done